src/EditorFeatures/CSharp/CompleteStatement/CompleteStatementCommandHandler.cs

```diff
@@ -392,8 +392,12 @@ private static bool SemicolonIsMissing(SyntaxNode currentNode)
                     return ((ExternAliasDirectiveSyntax)currentNode).SemicolonToken.IsMissing;
                 case SyntaxKind.ClassDeclaration:
                     return ((ClassDeclarationSyntax)currentNode).SemicolonToken.IsMissing;
+                case SyntaxKind.RecordDeclaration:
+                    return ((RecordDeclarationSyntax)currentNode).SemicolonToken.IsMissing;
```

Comment:
:memo: I did a search for usage of `StructDeclaration` and `IsRecord` and added cases for records and record structs if missing.
But some of them I was not able to cover with test. I'll call them out. This is one of them.

Reply:
@jcouv This private method is unused. That's why it's not covered by a test.

I had a PR in the past to remove it, but it hadn't got merged yet.